bazel/glog.bzl

```diff
@@ -93,13 +93,13 @@ def glog_library(namespace = "google", with_gflags = 1, **kwargs):
     ]
 
     windows_only_copts = [
+        "-DGLOG_NO_ABBREVIATED_SEVERITIES",
```

Comment:
This seems like it could break users. Can you add a #define to port.h instead, along the lines of the docs? https://github.com/google/glog#notes-for-windows-users

Reply:
On second thoughts, as it's in copts rather than defines, it's unlike to affect users but it would still be cleaner to use the approach in the docs IMO.

Reply:
Are you saying that the define is transitive, i.e., it is forwarded to library's consumer? Either way, CMake always defines `GLOG_NO_ABBREVIATED_SEVERITIES` on Windows to avoid conflicts due to `ERROR`:

https://github.com/google/glog/blob/a79416bab739907a31a1e139540921ff0492f010/CMakeLists.txt#L598-L600

If I recall correctly, on Windows glog without `GLOG_NO_ABBREVIATED_SEVERITIES` is unusable anyway. Regardless, we should probably sync the approaches taken by both build systems.

Reply:
defines are transitive (useful when needed in public headers) but copts are not. If CMake does it too then this LGTM, thanks!